router/transformer/transformer.go

```diff
@@ -279,6 +248,50 @@ func (trans *HandleT) Setup() {
 	trans.transformerResponseTransformRequestTime = stats.NewStat("router.transformer_response_transform_time", stats.TimerType)
 }
 
+func (trans *HandleT) makeHTTPRequest(ctx context.Context, url string, payload *strings.Reader) ([]byte, int, error) {
+	var respData []byte
+	var respCode int
+	payload.Seek(0, io.SeekStart)
+
+	req, err := http.NewRequestWithContext(ctx, "POST", url, payload)
```

Comment:
```suggestion
	req, err := http.NewRequestWithContext(ctx, "POST", url, io.NewReader(payload))
```

Reply:
I am not able to get the `NewReader` method from `io` package.  I am able to use `bytes.NewReader()` which is working in expected way

```
req, err := http.NewRequestWithContext(ctx, "POST", url, bytes.NewReader(payload))
```